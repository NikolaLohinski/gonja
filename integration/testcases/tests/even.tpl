{{ 2 is even }}
{{ 1 is even }}
{{ 0 is even }}
{{ 2.1 is even }}
{{ 'test' is even }}
