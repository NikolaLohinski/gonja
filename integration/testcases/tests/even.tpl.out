True
False
True
False
False
